app/lib/actions/katello/repository/update.rb

```diff
@@ -4,22 +4,36 @@ module Repository
       class Update < Actions::EntryAction
         middleware.use Actions::Middleware::KeepCurrentUser
 
+        # rubocop:disable Metrics/MethodLength
         def plan(repository, repo_params)
           action_subject repository
           repository = repository.reload
           repo_params[:url] = nil if repo_params[:url] == ''
           repository.update_attributes!(repo_params)
 
           if update_content?(repository)
+            content_url = ::Katello::Glue::Pulp::Repos.custom_content_path(repository.product, repository.label)
+
             plan_action(::Actions::Candlepin::Product::ContentUpdate,
                         :owner => repository.organization.label,
                         :content_id => repository.content_id,
                         :name => repository.content.name,
-                        :content_url => ::Katello::Glue::Pulp::Repos.custom_content_path(repository.product, repository.label),
+                        :content_url => content_url,
                         :gpg_key_url => repository.yum_gpg_key_url,
                         :label => repository.content.label,
                         :type => repository.content_type,
                         :arches => repository.arch == "noarch" ? nil : repository.arch)
+
+            katello_content_id = repository.product.product_content_by_id(repository.content_id).content_id
+            plan_action(::Actions::Katello::Content::Update,
+                        katello_content_id: katello_content_id,
+                        content_attrs: {
+                          name: repository.content.name,
+                          content_url: content_url,
+                          content_type: repository.content_type,
+                          label: repository.content.label,
+                          gpg_url: repository.yum_gpg_key_url})
```

Comment:
(I'm going to repeat this a few times), but i don't think we should abstract this AR update into a dynflow action